contrib/experimental/great_expectations_experimental/expectations/expect_column_values_number_of_decimal_places_to_equal.py

```diff
@@ -149,7 +149,7 @@ class ExpectColumnValuesNumberOfDecimalPlacesToEqual(ColumnMapExpectation):
 #     @classmethod
 #     @renderer(renderer_type="renderer.question")
 #     def _question_renderer(
-#         cls, configuration, result=None, language=None, runtime_configuration=None
+#         cls, configuration, result=None, runtime_configuration=None
```

Comment:
Some of these changes are in commented out code. Do we need to maintain these (ie can they be deleted)? Are these to provide examples?